File: Hari02/C++/Lingkaran.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main()
{
    double radius, luas, keliling;
    cout << "Masukkan jari-jari Lingkaran: ";
    cin >> radius;

    luas = M_PI * pow(radius, 2);
    keliling = 2 * M_PI * radius;

    cout << " Luas : " << luas << endl;
    cout << "Keliling : " << keliling << endl;
    return 0;
}

File: Hari02/Dart/Lingkaran.dart
import 'dart:math';

void main() {
  print("Masukkan jari-jari lingkaran:");
  double radius = double.parse(stdin.readLineSync()!);

  double luas = pi * pow(radius, 2);
  double keliling = 2 * pi * radius;

  print("Luas: $luas");
  print("Keliling: $keliling");
}
